src/T2DMap.cpp

```diff
@@ -2319,7 +2319,7 @@ void T2DMap::paintMapInfo(const QElapsedTimer& renderTimer, QPainter& painter, c
 
     // Left margin for info widget:
     uint infoLeftSideAvoid = 10;
-    if (mMultiSelectionListWidget.isVisible()) {
+    if (mMultiSelection && mMultiSelectionListWidget.isVisible()) {
```

Comment:
:-1: This seems wrong to me - `(bool) T2DMap::mMultiSelection` is a flag that is set whilst the drag to select rectangle is visible - but that is only when it is being dragged - **on the 2D map**! Using it here will balls-up the resizing of the room info display so that it will get overlaid by the room list widget once the drag to select rectangle disappears.

Reply:
Please demonstrate that `mMultiSelectionListWidget` is malformed at the point where you say it crashed...

Reply:
BTW It is hidden by the `T2DMap` constructor and should only be showing if `mMultiSelectionSet.size()` is greater than 1.

Reply:
See https://user-images.githubusercontent.com/110988/105327936-cd8f2380-5bcf-11eb-88f8-a417341cabc9.png where there is an obvious crash. Just fixing it is all.

Reply:
Yeah - but what I was trying to say is what is `mMultiSelectionListWidget` there? It isn't showing in the debugger view is what I was trying to get across and I thought you'd be able to repeat the crash and show what on earth has gone wrong to mess the value of `infoLeftSideAvoid` so much - given that it is calculated at the point of the crash from two characteristics of the widget.

Reply:
It's not happening again, but it stopped happening once I put this in. This is all we have to go off and as far as I see this fix does not do damage because `mMultiSelection` should be in sync with `mMultiSelectionListWidget.isVisible()` anyway.

Reply:
Not quite - *after* the selection rectangle has been used to grab a load of rooms then `mMultiSelection` will go back to being `false` but at that point the `mMultiSelectionListWidget` will (still) be visible and we need the `if` to stay `true` so that the info window does not disappear behind the widget showing all the selected rooms. That is why I don't think you can use `mMultiSelection` in this manner.

As I said before i really want to work out why `mMultiSelectionListWidget` does not appear to be valid (not constructed?) - which is why I want to try and reproduce it or at least try and see what sort of state it is in when this crash is happening. Until we can reproduce the crash I am just scratching my head - so how can I cause this to happen?

Reply:
Just opening the map did it before for me while testing the override map info PR.

Reply:
You have a fair point about getting to the bottom of the problem before, sorry I was frustrated before.

Reply:
So is this problem showing in the main `development` branch code or @Delwing 's draft?